A3A/addons/core/functions/Base/fn_destroyCity.sqf

```diff
@@ -1,11 +1,15 @@
-params ["_markerX"];
-
 private _positionX = getMarkerPos _markerX;
 private _size = [_markerX] call A3A_fnc_sizeMarker;
-
+private _buildings = _positionX nearobjects ["house",_size];
 private _buildings = _positionX nearObjects ["house",_size];
 
 {
+	if (random 100 < 70) then {
+		for "_i" from 1 to 7 do {
+			_x setHit [format ["dam%1",_i],1];
+			_x setHit [format ["dam %1",_i],1];
+		};
+	};
```

Comment:
Why is this necessary? Spamming setHit across the network isn't a great idea

Reply:
oh, this one isn't mine actually, don't know where it came from

Reply:
or maybe it is and it's not from #187

Reply:
Might aswell just remove this to be honest